src/library/dlganalysis.cpp

```diff
@@ -62,6 +62,13 @@ DlgAnalysis::DlgAnalysis(QWidget* parent,
             SIGNAL(selectionChanged(const QItemSelection &, const QItemSelection&)),
             this,
             SLOT(tableSelectionChanged(const QItemSelection &, const QItemSelection&)));
+
+    connect(pLibrary, SIGNAL(setTrackTableFont(QFont)),
```

Comment:
is this an unrelated change?  I still don't see the correct font being used in the analyze view of the library.  (That's been broken for a long time)